packages/design-system-docs/src/pages/startup/theming.md

```diff
@@ -38,7 +38,31 @@ Sass variables are documented on the relevant documentation pages, and are defin
 - [Colors]({{root}}/styles/color/)
 - [Spacing]({{root}}/styles/spacing/)
 - [Type sizes]({{root}}/styles/typography/)
-- You can also [browse all Sass variable files on GitHub](https://github.com/CMSgov/design-system/tree/master/packages/design-system/src/styles/settings/variables).
+
+You can also [browse all Sass variable files on GitHub](https://github.com/CMSgov/design-system/tree/master/packages/design-system/src/styles/settings/variables).
+
+#### Focus settings
+
+<table class="ds-c-table ds-c-table--borderless docs_table">
```

Comment:
I really like using the table for CSS variables and other data, but I think this is the first time we use it for build settings. I think it only makes sense if there are multiple entries in the table, or if we include the other build settings too like `$ds-include-base-html-rulesets`, `$font-path`, and `$image-path`. However, this would require rethinking how our docs are organized and it would be better suited for a `Build settings` section.

I think for now it would be easier for us to replace this table with an explanation, and a list of all the focus variables and mixins.

Reply:
For example, we randomly list the focus variables [here](https://github.com/CMSgov/design-system/pull/510/files#diff-1a9408d59d6706d79301b86f714aa077L54). I think we should move that from `Textfield.docs.scss` to here